File: backend/album-collection/Models/Song.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace album_collection.Models
{
    public class Song
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public double Duration { get; set; }
        public int AlbumId { get; set; }
        public virtual Album Album { get; set; }
        
        public Song()
        {

        }
        

    }
}

File: frontend/src/js/Components/Song.js
export default {
    DisplaySong,
}

function DisplaySong(song){
    return `
        <h1><label><strong>Song Title: </strong></label>${song.title}</h1>
        
        <h4><label>Song Duration: </label>${song.duration}</h4>

        <h4><label>Album: </label>${song.album.title}</h4>
        
    `;
}